broker/src/test/java/io/zeebe/broker/system/configuration/BrokerCfgTest.java

```diff
@@ -480,6 +480,42 @@ public void shouldSetBackpressureConfig() {
     assertThat(backpressure.getAlgorithm()).isEqualTo(LimitAlgorithm.GRADIENT);
   }
 
+  @Test
+  public void shouldUseConfiguredBackpressureAlgorithms() {
```

Comment:
Please move this test method into a dedicated class `BackPressureCfgTest`. 

This test class is already long enough. It tests a bunch of stuff, whereas you have a very focused and clean test